slides/src/content/component.md

```diff
@@ -226,13 +228,13 @@ export class Child {
 
 ## Quiz (continued)
 
-1. Parent:  
```

Comment:
1. This change seems to have come in by accident (unrelated to the main change of this PR).
2. The end-of-line spaces are to force line breaks - please use `<br>` instead of just removing them.